core/src/test/java/org/bitcoinj/params/AbstractBitcoinNetParamsTest.java

```diff
@@ -58,4 +55,16 @@ public void getBlockInflation() {
         assertEquals(Coin.FIFTY_COINS.div(2), BITCOIN_PARAMS.getBlockInflation(210000));
         assertEquals(Coin.FIFTY_COINS.div(2), BITCOIN_PARAMS.getBlockInflation(210001));
     }
+
+    static class AbstractBitcoinNetParamsSubClass extends AbstractBitcoinNetParams {
```

Comment:
Is there a better name? Maybe `AbstractBitcoinNetParamsEasiest` or `AbstractBitcoinNetParamsForTesting`?

Reply:
How about `MinimalAbstractBitcoinNetParams`?